src/MOI_wrapper.jl

```diff
@@ -260,20 +260,20 @@ end
 # coef: List of corresponding coefficients
 # d: dimension of set
 # rev: if true, we unscale instead (e.g. divide by √2 instead of multiply for PSD cone)
-function _scalecoef(rows, coef, d, rev)
+function _scalecoef(rows::AbstractVector{<: Integer}, coef::Vector{<: AbstractFloat}, d::Integer, rev::Bool)
     scaling = rev ? 1 / √2 : 1 * √2
-    output = copy(coef)
-    diagidx = BitSet()
-    for i in 1:d
-        push!(diagidx, trimap(i, i))
-    end
+    output = deepcopy(coef)
```

Comment:
I would do `coef::Vector{Float64}` and use `copy` instead of `deepcopy` as it is faster.